### PR TITLE
Add Edit-bConnectJob.ps1

### PR DESCRIPTION
**Pull request by kingkex**
Ich habe die Funktionen um Edit-bConnectJob.ps1 erweitert und mich dabei an die Form der anderen Edit-x gehalten. Es war eine Änderung an Invoke-bConnectPatch (IgnoreAssignments) notwendig, um auch Jobs ändern zu können, die bereits Endpoints zugewiesen haben.

### CODE REVIEW

File: bConnect/Public/Edit-bConnectJob.ps1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    Param (
+        [Parameter(Mandatory=$true)][PSCustomObject]$Job,
+        [boolean]$IgnoreAssignments = $false
+
+    )
+
+    $_connectVersion = Get-bConnectVersion
+    If($_connectVersion -ge "1.0") {
+        $_Job = ConvertTo-Hashtable $Job
+        return Invoke-bConnectPatch -Controller "Jobs" -Version $_connectVersion -objectGuid $_Job.Id -Data $_Job -IgnoreAssignments $IgnoreAssignments
```
Comment (T0biii) [inline bConnect/Public/Edit-bConnectJob.ps1:21]: maybe add Test-Guid
as in: 
https://github.com/baramundisoftware/PS-bConnect/blob/6f8b4c1abf4de1b0b2f42fa354b588c4f846d229/bConnect/Public/Edit-bConnectDynamicGroup.ps1#L18
https://github.com/baramundisoftware/PS-bConnect/blob/6f8b4c1abf4de1b0b2f42fa354b588c4f846d229/bConnect/Public/Edit-bConnectEndpoint.ps1#L18
https://github.com/baramundisoftware/PS-bConnect/blob/6f8b4c1abf4de1b0b2f42fa354b588c4f846d229/bConnect/Public/Edit-bConnectOrgUnit.ps1#L18
https://github.com/baramundisoftware/PS-bConnect/blob/6f8b4c1abf4de1b0b2f42fa354b588c4f846d229/bConnect/Public/Edit-bConnectStaticGroup.ps1#L18
https://github.com/baramundisoftware/PS-bConnect/blob/6f8b4c1abf4de1b0b2f42fa354b588c4f846d229/bConnect/Public/Edit-bConnectVariableDefinition.ps1#L18

**Comment by kingkex** [comment]
Your right about the inconsistency. I would like to see the result of your question #16, before I would make any change.

**Comment by T0biii** [comment]
@kingkex See: https://github.com/baramundisoftware/PS-bConnect/issues/16#issuecomment-634550891

**Comment by kingkex** [comment]
> @kingkex See: [#16 (comment)](https://github.com/baramundisoftware/PS-bConnect/issues/16#issuecomment-634550891)

i added the necessary change in order to resolve the inconsistency.

**Pull request merged**